dotCMS/src/main/java/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -648,6 +653,23 @@ else if(cont.getStructure().getStructureType()==Structure.STRUCTURE_TYPE_HTMLPAG
 			    pageMap.put("mimeType", "application/dotpage");
 	            pageMap.put("pageURI", ident.getURI());
 	            return pageMap;
+			} else if(cont.getStructure().getStructureType()== BaseContentType.DOTASSET.getType()) {
+
+				final java.io.File file = Try.of(()->cont.getBinary(DotAssetContentType.ASSET_FIELD_VAR)).getOrNull();
+				if (null != file) {
+					final String fileName = file.getName();
+					final String mimeType = servletContext.getMimeType(fileName.toLowerCase());
+
+					final Map<String, Object> fileMap = cont.getMap();
+					fileMap.put("mimeType", mimeType);
+					fileMap.put("path",          "/dA/" + cont.getIdentifier() + StringPool.SLASH);
+					fileMap.put("type", "dotasset");
+					fileMap.put("type", "dotasset");
+					fileMap.put("name",     fileName);
+					fileMap.put("fileName", fileName);
+
+					return fileMap;
```

Comment:
Don't we have a transformer you can use for this @jdotcms ?

Reply:
if not exists any, Should we create a new transformer?